tests/data/Modelica.mo

```diff
@@ -0,0 +1,21 @@
+// 21 lines 13 code 4 comments 3 blanks
```

Comment:
```suggestion
// 21 lines 13 code 5 comments 3 blanks
```

Guess, the header line counts as comment?

Reply:
I'm unsure. I looked at other examples and some do and some don't.

Reply:
Just guessed this based on the failure: 4 != 5.

Reply:
Well I guess it is worth a try.

Reply:
Looks like this was it. Thanks @beutlich